src/plugins/jsx/index.js

```diff
@@ -37,6 +37,19 @@ tt.jsxTagEnd.updateContext = function(prevType) {
   }
 };
 
+function isFragment(
+  object:
+    | ?N.JSXOpeningElement
+    | ?N.JSXClosingElement
+    | ?N.JSXOpeningFragment
+    | ?N.JSXClosingFragment,
+): boolean {
```

Comment:
Does Flow support type narrowing similarly to TypeScript? If it does, the return type could be something similar to `: object is N.JSXOpeningFragment | N.JSXClosingFragment`.

Reply:
Not that I can find in the docs! :(